test/controllers/actions/AuthActionImplSpec.scala

```diff
@@ -19,16 +19,17 @@ package controllers.actions
 import base.SpecBase
 import mocks.auth.MockMicroserviceAuthorisedFunctions
 import play.api.mvc.Results._
-import play.api.mvc.{Action, AnyContent}
+import play.api.mvc.{Action, AnyContent, ControllerComponents}
 import play.api.test.Helpers._
 import uk.gov.hmrc.auth.core._
+import play.api.test.Helpers.stubControllerComponents
 
 import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
I don't think you need this global execution context here unless there's something I'm missing?